### DISCUSSION TITLE
[Feature] Adding duplicate image to album should add image from respository

### DISCUSSION THREAD

**Discussion by chaddm** [immich-app/immich#13723] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### Feature 

When drag-n-drop is used to put images into an album, images that are de-duplicated will cause the target image to be added to the album.

### Workflow

I have a large group of images where I would like to drag-n-drop certain sets of images into different albums.  If I drag-n-drop images into an album, the duplicate detector will ignore uploads of duplicate items (which is awesome); however, the final set of images in the album does not include the target of the deduplicated assets.  I would expect the deduplication target image to be added along with the other images into the album so that everything I had in the drag-n-drop appears in the album.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by CrushedAsian255** [comment]
Hang on, let me just understand this.
Let's say your Immich Library contained {A,B,C,D}
You have an album with {B,C}
You drag-n-drop images {D,E}
The album should now contain {B,C,D,E} and your entire library should now contain {A,B,C,D,E}
Is this correct?

**Reply by wojciech-dron** [reply]
I think this is other situation:
Immich library contains {A,B}
I drag & drop {A,B,C,D} into an album.
This album contains only {C,D} because {A,B} were skipped.

Album should contain {A,B} from Immich library and {C,D} .

**Reply by Quenom** [reply]
As well as {C,D} that were uploaded so
{A,B,C,D}

**Reply by wojciech-dron** [reply]
Yes, I've updated my prevoius comment.

**Comment by Quenom** [comment]
NVM i may be stupid, it already works that way, i had photos in trash, and duplicates of those wouldnt get added to album

<details><summary>edited out</summary>
<p>
I would also say this classifies as a bug since image duplication check results in unwanted behavior(images not being added to albums)
(i expected that duplicates detected in upload to album means that these files were already in album, not somewhere else in a library)
In this situation i would prefer to allow duplicate images in library, and deal with them in it's utility.
Hunting down hundreds of photos all over library to add to album, isn't acceptable.
Perhaps a toggle in upload window setting to let through duplicates for single upload instance as a simple workaround?
![image](https://github.com/user-attachments/assets/d3d94059-f019-4173-b4fd-34a9f8294c66)
</p>
</details>

**Comment by bo0tzz** [comment]
No idea when it was implemented but this currently works as expected.
